libs/net/models/Areas/Services/Models/Content/ContentModel.cs

```diff
@@ -121,6 +121,11 @@ public class ContentModel : AuditColumnsModel
     /// </summary>
     public bool IsApproved { get; set; }
 
+    /// <summary>
+    /// get/set - The product.
+    /// </summary>
+    public ProductModel Product { get; set; } 
```

Comment:
Need to specify nullability `public ProductModel? Product { get; set; }`

Reply:
Updated, thanks